helm_deploy/hmpps-interventions-service/reports/crs_performance_report.sql

```diff
@@ -26,9 +26,10 @@ COPY (
     ap.submitted_at         AS date_first_action_plan_submitted,
     'coming-later'          AS date_of_first_action_plan_approval,
     (
-      select min(app.appointment_time)
-      from action_plan_appointment app
-      where app.action_plan_id = ap.id and attended in ('YES', 'LATE')
+        select min(app.appointment_time)
```

Comment:
This sql is currently failing and needs to be re-looked at. It was updated on main since my intitial change